readme.md

```diff
@@ -51,6 +51,13 @@ const execa = require('execa');
 	const {stdout} = await execa.shell('echo unicorns');
 	//=> 'unicorns'
 
+	// Cancelling a spawned process
+	const spawned = execa("echo unicorns");
+	spawned.catch( error => {
+		// This would be equal to the reason provided for canceling
```

Comment:
This comment should probably be removed because `error.message` is actually changed by `execa` so `error.message` would not be exactly `"reason for canceling"`.